app/src/App.vue

```diff
@@ -0,0 +1,199 @@
+                                <span v-else class="no-link">Please select a product and an action</span>
+                            </el-form-item>
+                        </el-form>
+                    </el-col>
+                </el-row>
+            </PsCardBlock>
+
+            <PsCardFooter>
+                <template slot="footerRight">
+                    <el-button type="primary" @click="validateForm('form')">{{ translations.copyToClipboard }}</el-button>
+                </template>
+            </PsCardFooter>
+        </PsCard>
+
+        <BannerPromo
+            :id_product_addons="26835"
+            :iso_code="context.country.iso_code"
+            :iso_lang="context.language.iso_code"
+            :ps_version="ps_version"
+            :trackingAddonsLink="trackingAddonsLink"
+        >
+            <el-row>
+                <el-col :span="12">
+                    <span class="features"><i class="material-icons">add_circle</i> Card & Button design integration</span>
+                </el-col>
+                <el-col :span="12">
+                    <span class="features"><i class="material-icons">add_circle</i> Overview and Card Manager</span>
+                </el-col>
+            </el-row>
+            <el-row>
+                <el-col :span="12">
+                    <span class="features"><i class="material-icons">add_circle</i> Statistics</span>
+                </el-col>
+                <el-col :span="12">
+                    <span class="features"><i class="material-icons">add_circle</i> Virtual cart on external pages</span>
+                </el-col>
+            </el-row>
+        </BannerPromo>
+    </div>
+</template>
+
+<script>
+import axios from 'axios'
+import PsCard from '@/components/PsCard.vue'
+import PsCardHeader from '@/components/PsCardHeader.vue'
+import PsCardBlock from '@/components/PsCardBlock.vue'
+import PsCardFooter from '@/components/PsCardFooter.vue'
+import BannerPromo from '@/components/BannerPromo.vue'
+
+const shopContext = JSON.parse(context)
+const translations = JSON.parse(confTranslations)
+
+export default {
+    name: 'configuration',
+    components: {
+        BannerPromo,
+        PsCard,
+        PsCardHeader,
+        PsCardBlock,
+        PsCardFooter
+    },
+    data: function () {
+        return {
+            product: '',
+            form: {
+                selectedProduct: '',
+                selectedAction: ''
+            },
+            rulesForm: {
+                selectedProduct: [
+                    { required: true, message: 'Please select a product', trigger: 'change' }
+                ],
+                selectedAction: [
+                    { required: true, message: 'Please select an action', trigger: 'change' }
+                ]
+            },
+            generatedLink: 'index.php?fc=module&module=ps_buybuttonlite&controller=RedirectManager&idproduct',
```

Comment:
This string `/index.php?fc=module&module=ps_buybuttonlite&controller=RedirectManager` is repeated a few lines below. It looks like a good candidate for a function like `getLinkFor...()`

Reply:
Hm ... I forgot to change this. Solved, I generate the link with getModuleLink() in php and send it to js.